packages/playground/index.html

```diff
@@ -4,183 +4,10 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/fp-logo-fav.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Fiberplane Playground</title>
+    <title>(Local) Fiberplane Playground</title>
   </head>
   <body>
     <div id="root"></div>
     <script type="module" src="/src/main.tsx"></script>
-    <script>
```

Comment:
we want to rely on vite to inject this code, so should remove it from `index.html`